app/src/main/java/de/westnordost/streetcomplete/settings/OAuthFragment.kt

```diff
@@ -66,7 +66,7 @@ class OAuthFragment : Fragment(R.layout.fragment_oauth), BackPressedListener, Ha
             authorizeUrl = inState.getString(AUTHORIZE_URL)
             oAuthVerifier = inState.getString(OAUTH_VERIFIER)
         } else {
-            consumer = okHttpOAuthConsumer()
+            consumer = get()
```

Comment:
I'd prefer it to be explicit/expressive even if it is not necessary

```suggestion
            consumer = get<OAuthConsumer>()
```